src/System.Composition.Hosting/src/System/Composition/Hosting/Providers/ExportFactory/ExportFactoryExportDescriptorProvider.cs

```diff
@@ -11,7 +11,7 @@
 
 namespace System.Composition.Hosting.Providers.ExportFactory
 {
-    internal class ExportFactoryExportDescriptorProvider : ExportDescriptorProvider
+    public class ExportFactoryExportDescriptorProvider : ExportDescriptorProvider
```

Comment:
This change is introducing a new public APIs, and so it would need to go through new API review process.